rest/taskana-rest-spring/src/main/java/pro/taskana/rest/workbasket/WorkbasketDefinitionController.java

```diff
@@ -159,10 +160,8 @@
       boolean noWrongAuth =
           definition.getAuthorizations().stream()
               .noneMatch(
-                  access -> {
-                    return (!access.getWorkbasketId().equals(importedWb.getId()))
-                        || (!access.getWorkbasketKey().equals(importedWb.getKey()));
-                  });
+                  access -> (!access.getWorkbasketId().equals(importedWb.getId()))
+                                || (!access.getWorkbasketKey().equals(importedWb.getKey())));
       if (!noWrongAuth) {
```

Comment:
Can you please further simplify this:
an inverse as a variable name is not clean.
- rename the variable to `authenticated`
- replace noneMatch with anymatch
- invert predicate

Reply:
The error message below has to be adjusted too then :)